src/pyclaw/sharpclaw/solver.py

```diff
@@ -343,6 +375,40 @@ def step(self,solution,take_one_step,tstart,tend):
             for j in range(num_stages):
                 state.q += self.b[j]*self._registers[j].q
 
+        elif self.time_integrator == 'DWRK':
+            # General explicit downwind RK with specified coefficients
+            # Implementation is in the Shu-Osher form
+
```

Comment:
It might be worthwhile to put the formulas here in a comment, to aid future debugging.